README

```diff
@@ -173,6 +173,7 @@ CHECK IDS
 	W-009: Module name does not match file name
 	W-010: Call to unknown interface
 	W-011: Declaration in require block not defined in own module
+	W-012: Usage of unknown permission macro
```

Comment:
Why a warning rather than an error?  If this isn't a false positive, won't it always fail?

Reply:
Converted to an error; I wasn't confident enough it would produce no false-positives